### PR TITLE
Improve mail order message

### LINKED ISSUE #27581: Missing product name in template "order_customer_comment"
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In mail template "order_customer_comment" the {product_name} variable is not included in the content of the mail, while it's loaded by the controller. No way for the admin to know which product the customer is talking about.

Moreover, the mail say "Hello {firstname} {lastname}", but this email is sent to the admin of the shop and firstname and lastname are information about the customer... so it should just be "Hello"

### Expected behavior

Having the product name included in the content of the mail

### Steps to reproduce

1. from an order detail in front-end, choose a product and send a message about that product.
2. Check your mail as the shop administrator... the product name is not mentioned in that mail...

### PrestaShop version(s) where the bug happened

1.7.8.2

### PHP version(s) where the bug happened

7.3

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by Prestaplugins**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add the {product_name} to the content of the mail, it's already loaded by the controller but missing in the mail and improve wording
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #27581 .
| How to test?      | Send a mail from an order detail in front-end, check the message received by the admin
| Possible impacts? |
| Sponsor company   | Prestaplugins

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27643)

### CODE REVIEW

**Review by matthieu-rolland — COMMENTED** [review]
thank you @Prestaplugins !

lgtm

**Review by matthieu-rolland — APPROVED** [review]
(approved but waiting for wording review)

**Review by NeOMakinG — APPROVED** [review]
Looks fine to me, but waiting for @Julievrz review :p

**Review by Julievrz — COMMENTED** [review]
LGTM, thanks. But I couldn't see if this issue was resolved in this PR?
image

It should be "Hello," instead of "Hello %name of the customer%,"

**Comment by Prestaplugins** [comment]
> LGTM, thanks. But I couldn't see if this issue was resolved in this PR? image
> 
> It should be "Hello," instead of "Hello %name of the customer%,"

Hi Julie, it was already fixed, probably in another issue

**Review by Progi1984 — CHANGES_REQUESTED** [review]
Please make changes in the repo `mjml-theme-converter` : https://github.com/PrestaShop/mjml-theme-converter/blob/master/mails/themes/modern_mjml/core/order_customer_comment.mjml.twig

**Comment by kpodemski** [comment]
@Progi1984 to unblock this PR I'm putting "Waiting for QA", I'll handle external repo myself after the QA Approved

**Review by HanaRebaiQA — APPROVED** [review]
Hello @Prestaplugins 

I have checked this PR. The issue is fixed. Here my tested cases : 

- Check the automatic tests : OK 
- Check with different email-theme (modern, classic) : OK
- Check with maildev : OK
- Check the different combinations with different translations and different email-theme : OK
- Check with different languages (RTL and no RTL) : OK
- Check with multistore : OK
- Check with different browsers : OK 

For the RTL with modern email-theme, it should be fixed with mjml-converter

image
imageimage!
image
[image](https://user-images.githubusercontent.com/92912932/175040470-795fea23-c367-45ed-9b61-80fb85a764f7.png)
image
image


So, it's QA :heavy_check_mark: 

Thank You!

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @Prestaplugins & @HanaRebaiQA 

@kpodemski Ready for new PR :muscle:

**Pull request merged**
